lib/config/wrapper.go

```diff
@@ -431,3 +431,15 @@ func (w *Wrapper) StunServers() []string {
 
 	return addresses
 }
+
+func (w *Wrapper) MyName() string {
+	w.mut.Lock()
+	myID := w.cfg.MyID
+	w.mut.Unlock()
+	device, ok := w.Device(myID)
```

Comment:
I think just skip the comma0ok and return device.Name regardless. It should /always/ exist, and if it doesn't it's something really weird with the config that we'll fix on next startup so no need to tell the user something they can't do anything about.